package/com.unity.formats.usd/Editor/Scripts/Behaviors/UsdMenu.cs

```diff
@@ -14,9 +14,9 @@
 
 using System.IO;
 using System.Linq;
-using UnityEngine;
 using UnityEditor;
 using USD.NET;
+using Debug = UnityEngine.Debug;
```

Comment:
Not used?

Reply:
It's used in a function that I haven't changed, where previous it just used the UnityEngine include instead. I'll revert this change :)